dask_expr/tests/test_collection.py

```diff
@@ -98,6 +98,12 @@ def test_reductions(func, pdf, df):
     assert_eq(func(df.x), func(pdf.x))
 
 
+def test_nbytes(pdf, df):
+    with pytest.raises(NotImplementedError, match="nbytes is not implemented"):
+        df.nbytes
+    assert_eq(df.x.nbytes.compute(), pdf.x.nbytes)
```

Comment:
Well, actually, we should drop the `.compute()` I would expect `assert_eq` to handle this.

Reply:
Yikes, yes of course. Thx